tsdx.config.js

```diff
@@ -15,19 +15,26 @@ module.exports = {
       case 'umd':
         delete config.external
         config.output.indent = false
-        config.plugins.push(
+        config.plugins.unshift(
```

Comment:
The plugins were being called after TS compilation, and weren't behaving as desired. I've moved them to the front of the plugin pipeline so that the compiled code takes the modifications into account.